gpii/node_modules/canopyMatchMaker/test/CanopyMatchMakerTests.js

```diff
@@ -0,0 +1,473 @@
+/*
+* Canopy Match Maker Yrdyd
+*
+* Copyright 2015 Raising the Floor - International
+*
+* Licensed under the New BSD license. You may not use this file except in
+* compliance with this License.
+*
+* The research leading to these results has received funding from the European Union's
+* Seventh Framework Programme (FP7/2007-2013)
+* under grant agreement no. 289016.
+*
+* You may obtain a copy of the License at
+* https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+// TODO: This should be converted to a set of browser tests
+
+(function () {
+    "use strict";
+
+    var fluid = require("infusion"),
+        jqUnit = fluid.registerNamespace("jqUnit"),
+        kettle = require("kettle"),
+        gpii = fluid.registerNamespace("gpii");
+
+    kettle.loadTestingSupport();
+
+    require("canopyMatchMaker");
+    require("ontologyHandler"); // TODO: We don't require to depend on the entire ontologyHandler
+    require("matchMakerFramework");
+
+    fluid.registerNamespace("gpii.canopyMatchMaker.tests");
+
+    fluid.defaults("gpii.canopyMatchMaker.tests", {
+        gradeNames: ["fluid.test.testEnvironment", "autoInit"],
```

Comment:
I don't think making this a "testEnvironment" achieves much since none of these tests are asynchronous?